ui/src/app/layout/topics/topicmetadatatable/topic-metadata-table.component.html

```diff
@@ -203,6 +203,8 @@ <h4 class="modal-title">{{ 'View Topic Data' | translate }}</h4>
             <span aria-hidden="true">&times;</span>
         </button>
     </div>
+    <ngb-alert class="m-3" [innerHTML]="'NOTICE_TOPIC_DATA_VIEW' | translate" [dismissible]="false">
```

Comment:
You should explicitly specify the type of the alert (info, warning, danger) to make the code more readable here.